Lib/pathlib.py

```diff
@@ -403,8 +241,9 @@ class PurePath(object):
     """
     __slots__ = (
         '_drv', '_root', '_parts',
-        '_str', '_hash', '_pparts', '_cached_cparts',
+        '_str', '_hash', '_pparts', '_cached_ncparts',
```

Comment:
What are `_pparts` (path parts) and `_cached_ncparts` (UNC is the best I can think of)? Maybe a comment to explain what the attributes are expected to be used for?

Reply:
`cparts` meant "casefolded parts", and `ncparts` means "normcase()d parts". I'll add a comment.

Reply:
I've renamed them as follows:

- `_pparts` --> `_parts_tuple`
- `_ncparts` --> `_parts_normcase`
- `_cached_ncparts` --> `_parts_normcase_cached`